docs/model-serving/quickstart.md

```diff
@@ -0,0 +1,175 @@
+# Quickstart Guide
+
+The Hopsworks Model Serving provides operational capabilities on top of the Model Registry (see the [docs](../model-registry/quickstart.md)), for the deployment and operation of trained models.
+
+<!--
+A model is the product of training a machine learning algorithm with training data. It could be an image classifier used to detect objects in an image, such as for example detecting cancer in an MRI scan. -->
+
+In this Quickstart Guide we are going to focus on how to configure the deployment of an existing model in the Model Registry and start making predictions with it.
+
+
+### HSML library
+
+The Hopsworks Model Serving library is part of `hsml` (**H**opswork**s** **M**achine **L**earning).
+The library is Apache V2 licensed and available [here](https://github.com/logicalclocks/machine-learning-api). It currently comes with a Python SDK.
+If you want to connect to the Model Serving from outside Hopsworks, see our [integration guides](../integrations/setup.md).
+
+The library is build around metadata-objects, representing entities within Model Serving. You can modify metadata by changing it in the metadata-objects and subsequently persisting it to the Model Serving. In fact, the Model Serving itself is also represented by an object. Furthermore, these objects have methods to operate with the deployments in the Model Serving.
+
+
+### Guide Notebooks
+
+This guide is based on a [series of notebooks](https://github.com/logicalclocks/hops-examples/tree/master/notebooks/ml), which are available in the Deep Learning Demo Tour Project on Hopsworks.
+
+
+### Connection, Project and Model Serving
+
+The first step is to establish a connection with your Hopsworks Model Serving instance and retrieve the object that represents the Model Serving you'll be working with.
+
+By default `connection.get_model_serving()` returns the Model Serving of the project you are working with.
+
+=== "Python"
+
+    ```python
+    import hsml
+
+    # Create a connection
+    connection = hsml.connection()
+
+    # Get the model serving handle for the project's model serving
+    ms = connection.get_model_serving()
+    ```
+
+
+### Integration with the Model Registry
+
+The Model Serving library integrates smoothly with the Model Registry library. Metadata objects such as [Model](../generated/model-registry/model.md) can be used in multiple methods in the Model Serving library to simplify the code and avoid duplication of parameters.
+
+As a result, the easiest way to deploy an existing model is using the `model.deploy()` method available in the model metadata object. For instance, the version with highest accuracy of a trained model can be deployed with the following lines.
```

Comment:
is using the -> is to call `model.deploy()`.
No need for this: "method available in the model metadata object."